submissions/HannaSyn/friends-app/index.js

```diff
@@ -0,0 +1,128 @@
+const searchInput = document.querySelector('.search-input');
+const sortAge = document.querySelector('.sort-age');
+const sortName = document.querySelector('.sort-name');
+const mainContent = document.querySelector('.main-content');
+const filterGender = document.querySelector('.gender-filter');
+const ageAscending = document.getElementById('age-ascending');
+const ageDescending = document.getElementById('age-descending');
+const nameAscending = document.getElementById('name-ascending');
+const nameDescending = document.getElementById('name-descending');
+const url = 'https://randomuser.me/api/?results=30';
+let users = [];
+
+const loadUsers = async () => {
+  try {
+    const res = await fetch(url);
+    const { results } = await res.json();
+    users = results;
+  }
+  catch(error) {
+    mainContent.innerHTML = `<p>Oops! Here is the error ${error}</p>`;
+  }
+}
+
+function renderCards(arr) {
+  let card = '';
+  
+  arr.forEach(({name, picture, dob, email}) => {
+    const template = `<div class="card">
+      <div class="card__content">
+        <figure class="card__photo">
+          <img src=${picture.large} alt="photo">
+        </figure>
+        <div class="card__info">
+          <p class="card__name">${name.first} ${name.last}</p>
+          <p class="card__age">${dob.age} y.o.</p>
+          <a href="mailto:${email}" class="card__email">${email}</a>
+        </div>
+      </div>
+    </div>`
+    card += template;
+  })
+  mainContent.innerHTML = card;
+}
+
+function filterBySearch(arr, str) {
+  return arr.filter( el => `${el.name.first}${el.name.last}`.toLowerCase().includes(str.toLowerCase()));
+}
+
+function filterByGender(value, arr) {
+  if (value === 'all') {
+    return arr;
+  } 
+  return arr.filter(el => el.gender === value);
+}
+
+function findChecked(element) {
+  let checked;
+  const inputs = element.querySelectorAll('input');
+  for (let input of inputs) {
+    if (input.checked) {
+      checked = input;   
+    }
+  }
+  return checked;
+  
+}
+
+function sortByNameAsc(arr) {
```

Comment:
You can also use arrow functions in these functions. It can help to read you code easily.
```suggestion
const sortByNameAsc = arr => arr.sort((a,b) => a.name.first.toLowerCase() < b.name.first.toLowerCase() ? -1 : 1)
```